_docs/services/app-autoscaler.md

```diff
@@ -0,0 +1,472 @@
+---
+parent: services
+layout: docs
+sidenav: true
+title: App-AutoScaler
```

Comment:
```suggestion
title: Application AutoScaler
```

This title will actually be displayed on the page and in the side navigation, so it should be full words without any abbreviation.